@ecomplus/storefront-template/template/scss/themes/thais-baby/_variables.scss

```
@@ -0,0 +1,4 @@
+:root {
+    --pink: #fd9da2 !important;
+    --indigo: #e6f9f8 !important;
+}
```

Melhor usarmos as variáveis SCSS em vez de CSS aqui (na verdade as CSS vars serão geradas a partir das variáveis SCSS também):

```suggestion
$pink: #fd9da2 !default;
$indigo: #e6f9f8 !default;
```